src/utils/localStorageOrders.ts

```diff
@@ -130,9 +131,6 @@ export function confirmOrderSubmission(chainId: number, account: string, submiss
 
 export const getUniqueOrders = (allOrders: Order[]): Order[] => [
   ...new Map(
-    allOrders
-      // sort by `updatedAt` asc so that the most recent one will be used
-      .sort((a, b) => parseFloat(a.updatedAt) - parseFloat(b.updatedAt))
-      .map((order) => [order.id, order]),
+    orderBy(allOrders, (order) => parseFloat(order.updatedAt), 'desc').map((order) => [order.id, order]),
```

Comment:
@Chef-Cheems To get the most recent one shouldn't we order by descending?

Reply:
yes, descending. idk why it was asc and why nobody noticed 🤔 
thanks!